components/base/FilterTabs.tsx

```diff
@@ -43,7 +43,7 @@ export const TabStyle = styled.div`
 const SelectWrapper = styled.div`
   width: 150px;
   padding-bottom: 10px;
-  @media (${minWidth(BREAKPOINT_MOBILE)}) {
+  @media (${breakUp("sm")}) {
```

Comment:
This applies a breakpoint upwards, with the rule starting to apply on the "sm" screen size.